website/settings/development.py

```diff
@@ -2,17 +2,6 @@
 
 ALLOWED_HOSTS = []
 
-DATABASES = {
```

Comment:
Thanks :) I forgot I added it when I was thinking about breaking out the test one (but I learned django already does that for us)